modules/API/API.js

```diff
@@ -743,6 +753,13 @@ function initCommands() {
                 return;
             }
 
+            if (transcription) {
```

Comment:
I'm wondering if the the recording have been started with transcription but it is stopped with transcription `false`, what would happen? The transcription will just continue?

Also what happens if the recording have been started without transcription but it is stopped with transcription `true`. Have you tested it?

Reply:
Yes, the transcription is allowed to continue in that case.